driver-core/src/main/com/mongodb/operation/ChangeStreamOperation.java

```diff
@@ -126,20 +126,22 @@ public FullDocument getFullDocument() {
      *
      * <p>A null value represents the server default.</p>
      *
-     * @return the resumeAfter
+     * @return the resumeAfter resumeToken
+     * @since 3.11
+     * @mongodb.server.release 4.2
      */
-    public BsonDocument getResumeToken() {
-        return resumeToken;
+    public BsonDocument getResumeAfter() {
```

Comment:
This is a breaking change to a deprecated but public api.  It may have been discussed and if so ignore, but if not can we deprecate the old `getResumeToken` method rather than just remove it?

Reply:
I have deprecated the `getResumeToken()` method.